recipes/jina-hubble-sdk/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "jina-hubble-sdk" %}
+{% set version = "0.2.0" %}
```

Comment:
Why such an old version? The latest is 0.23.

Reply:
Our repo `Jina` has dependencies on the old `jina-hubble-sdk` version, so how do we release the old `jina-hubble-sdk `version.

Reply:
Merging this PR will create the initial feedstock with 0.2.0 and packages will be built for it. After that, our auto-tick bot will issue a PR updating to the latest version.

Reply:
Thanks for your reply, and please merge this PR if has no other problem.

Reply:
> > our auto-tick bot will issue a PR updating to the latest version.

 What should I do if I want to publish all the old versions before (according to the version on pypi)？

Reply:
You can make manual PRs to the main branch updating to each version you want and avoid merging the automated PR until you're done getting older versions.

With that said, I'd strongly encourage you to consider whether you actually need all of those old versions.

Reply:
Yes, we actually need all of those old versions. thanks for your reply.